src/App.tsx

```diff
@@ -1,7 +1,8 @@
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import React, { useContext, useEffect } from 'react';
+import { registerRootComponent } from 'expo';
+import React, { useContext } from 'react';
```

Comment:
I'll add back useEffect() it's for auto firing the currently commented out tryLocalSignIn() function. Aka, if a users tokens are still valid they wont be forced to sign-in again when opening the app.

Reply:
Was it the linter that complained?

Reply:
nope
I left it commented out cuz in justin's pr you said to leave it commented out for now